py3status/modules/air_quality.py

```diff
@@ -45,14 +51,8 @@
     {time} epoch timestamp, eg 1510246800
     {time_s} local timestamp, eg 2017-11-09 17:00:00
     {time_tz} local timezone, eg -06:00
-
-    `{attribution_?_name}` can have more than 0.
-    `{attribution_?_url}` can have more than 0.
-
-    Your station may have individual scores for pollutants not listed below.
-    See https://api.waqi.info/feed/@UID/?token=TOKEN (Replace UID and TOKEN)
-    for an explicit list of valid placeholders to use, eg
-
+    {attribution_?_name} can have more than 0.
+    {attribution_?_url} can have more than 0.
```

Comment:
You could consolidate this with placeholders above. The `Note` description is for `iaqi_*` placeholders below.

Reply:
fixme @tobes and I merge